src/pages/Swap/index.tsx

```diff
@@ -86,6 +88,17 @@ export function getIsValidSwapQuote(
   return !!swapInputError && !!trade && (tradeState === TradeState.VALID || tradeState === TradeState.SYNCING)
 }
 
+export function largerPercentValue(a?: Percent, b?: Percent) {
+  if (a && b) {
+    return a.greaterThan(b) ? a : b
+  } else if (a) {
+    return a
+  } else if (b) {
+    return b
+  }
+  return undefined
+}
+
```

Comment:
nit: I feel kinda meh on this being a fn since it's p simple and only used once.

It shouldn't be exported. It's only used here, and linters won't usually alert you to unused variables if they're exported.

Reply:
agree, this function seems a bit strange to me

Reply:
removed the export. i felt like reading the code i had to think too much about what was going on, so it helped me to just name the function

Reply:
fair enough to me then